02-Reproject-Raster-In-R.Rmd

```diff
@@ -83,18 +91,23 @@ This lesson is a part of a series of raster data in R lessons:
 * [Lesson 06 - Plot Raster Time Series Data in R Using RasterVis and LevelPlot]({{ site.baseurl}}/R/Plot-Raster-Times-Series-Data-In-R/)
```

Comment:
My inclination now is to add them back into lesson 00.   (I'd forgotten I removed them).